assets/scss/modules/_forms.scss

```diff
@@ -16,45 +17,110 @@
   }
 }
 
-.content-checkbox {
-  width: 100%;
+.holder-item {
+  display: block;
   position: relative;
-  display: inline-block;
+  float: left;
+  box-sizing: border-box;
   text-align: left;
-  margin: 20px 0;
 
-  .checkbox {
-    display: flex;
+  &--small {
+    width: 50%;
+    margin-bottom: 30px;
```

Comment:
Great place to start using lvls (a variable that we can change after when santi have the real values)

Reply:
Yes! It's true, maybe in this case add the class for de margin